src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -68,6 +62,10 @@ def set_image
         flash[:error] = "KIWI image '#{params[:id]}' does not exist"
         redirect_back(fallback_location: root_path)
       end
+
+      def set_repositories
+        @repositories = @image.repositories.order(:order)
```

Comment:
Is this necessary? You can just call `@image.repositories` in the view? The order is already implemented on the association:

https://github.com/evanrolfe/open-build-service/blob/master/src/api/app/models/kiwi/image.rb#L13